<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>help_wl</class>
 <widget class="QWidget" name="help_wl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QLabel" name="labelURL">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">TextLabel</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../screencast.qrc">
         <normaloff>:/pictures/systray/exit.png</normaloff>:/pictures/systray/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../screencast.qrc"/>
 </resources>
 <connections/>
</ui>
